src/amo/components/AddonReviewListItem/index.js

```diff
@@ -234,6 +254,24 @@ export class AddonReviewListItemBase extends React.Component<InternalProps> {
                 ? i18n.gettext('Edit my reply')
                 : i18n.gettext('Edit my review')}
             </a>
+            <ConfirmButton
+              buttonType="cancel"
+              className={makeClassName(
+                'AddonReviewListItem-control',
+                confirmButtonClassName,
+              )}
+              id={confirmButtonClassName}
```

Comment:
I think there should only be 1 id on a page so maybe id can be concatenated with review id so they are unique?

Reply:
to clarify: should only have 1 id with the same name that is...

Reply:
Ah, good point!